test/operation.ts

```diff
@@ -143,9 +143,7 @@ describe('GrpcOperation', () => {
         done();
       });
       // tslint:disable-next-line no-any
-      (grpcOperation as any)
-        .poll_()
-        .then((r: r.Response) => {}, assert.ifError);
+      (grpcOperation as any).poll_().then((r: Response) => {}, assert.ifError);
```

Comment:
If it makes life easier
you could use `util.noop` instead of `(r: Response) => {}`